testsuite/forge-cli/src/suites/realistic_environment.rs

```diff
@@ -439,11 +439,11 @@ pub(crate) fn realistic_network_tuned_for_throughput_test() -> ForgeConfig {
                     }
                     OnChainExecutionConfig::V4(config_v4) => {
                         config_v4.block_gas_limit_type = BlockGasLimitType::NoLimit;
-                        config_v4.transaction_shuffler_type = TransactionShufflerType::SenderAwareV2(256);
+                        config_v4.transaction_shuffler_type = TransactionShufflerType::default_for_genesis();
```

Comment:
if it's default we don't need to override, right?

probably should overwrite with 
```
    UseCaseAware {
        sender_spread_factor: 256,
        platform_use_case_spread_factor: 0,
        user_use_case_spread_factor: 0,
    },
```